src/utils/smartParser.ts

```diff
@@ -174,7 +174,7 @@ export const smartParser = (
                       ) {
                         parsedDataPoint[avastarParsedDataPointProperties[k]] = (
                           parsingModel as any
-                        )[filePathModel]['None'][categorySelector][k][
+                        )[filePathModel][nestedArrayName][categorySelector][k][
```

Comment:
why was it none ?

Reply:
Aucune idée, en tout cas ce n'était pas correct.